frontend/src/pages/Notifications/NotificationsSnackbar.js

```diff
@@ -68,7 +73,7 @@ const NotificationsSnackbar = props => {
   return (
     <Snackbar open={props.showSnackbar} autoHideDuration={4000} onClose={props.closeSnackbar}>
       <SnackbarContentWrapper
-        variant={props.snackbarError ? "error" : "success"}
+        variant={props.snackbarError ? "error" : props.snackbarWarning ? "warning" : "success"}
```

Comment:
Use the ternary expression for one condition only since it would be difficult to read. Create own variable and refactor the ternary expression into a normal if condition.